source/transactions-convenient-api/tests/commit-writeconcernerror.yml

```diff
@@ -0,0 +1,196 @@
+database_name: &database_name "withTransaction-tests"
+collection_name: &collection_name "test"
+
+data: []
+
+tests:
+  -
+    description: commitTransaction is not retried after write concern error (WriteConcernFailed)
```

Comment:
My reading of the [transactions spec](https://github.com/mongodb/specifications/blob/master/source/transactions/transactions.rst#unknowntransactioncommitresult) is that this should be labelled as "UnknownTransactionCommitResult" and so be retried.

Reply:
If I understand correctly, you're saying that the driver's internal retry-once behavior (documented under [UnknownTransactionCommitResult](https://github.com/mongodb/specifications/blob/master/source/transactions/transactions.rst#unknowntransactioncommitresult)) will result in `commitTransaction` being retried and succeeding on the second attempt. I just added some more content to the original spec document to talk about this, so please give that a read.

I think the right thing to do here is to configure the fail point to occur twice, to allow control to return to `withTransaction` so we can rightfully decide _not_ to retry.

Reply:
Yes, that's what I'm saying.  Where should I look in the original spec document exactly?